stores/UserProfileStore.js

```diff
@@ -199,8 +211,8 @@ class UserProfileStore extends BaseStore {
     }
 
     fillInUserDecks(payload) {
-        this.userDecks = [];
```

Comment:
@rmeissn Do you remember why you dont just assigned the object/array? Did you experienced later changes because it was handled as reference?

Reply:
@schatzopoulos Please validate that assigning (this.userDecks = payload.decks;) does not result in a reference which caues problems. Such a reference could cause that e.g. this.userDecks could be changed by the action.